wdl/transforms/draft2/src/main/scala/wdl/transforms/draft2/wdlom2wom/WdlDraft2WomCommandTaskDefinitionMaker.scala

```diff
@@ -27,12 +28,19 @@ object WdlDraft2WomCommandTaskDefinitionMaker extends WomCommandTaskDefinitionMa
     // Figure out the start line of the workflow in the source file
     val t: Option[Terminal] = AstTools.findTerminals(wdlTask.ast).headOption
 
+    def stringifyMetaValues(meta: Map[String, String]): Map[String, MetaValueElement] = {
```

Comment:
I think it would be great to explain in a comment that draft-2 does not support composite meta value elements, only strings